website/docs/Queries.md

```diff
@@ -26,10 +26,9 @@ title: Queries
     - [Default state for: `disabled`, `selected`, and `busy` keys](#default-state-for-disabled-selected-and-busy-keys)
     - [Default state for: `checked` and `expanded` keys](#default-state-for-checked-and-expanded-keys)
   - [`ByA11Value`, `ByAccessibilityValue`](#bya11value-byaccessibilityvalue)
-- [Hidden accessibility option](#hidden-accessibility-option)
-  - [Examples](#examples)
-    - [`display: "none"`](#display-none)
-    - [`accessibilityElementsHidden`](#accessibilityelementshidden)
+- [Common query options](#common-query-options)
+  - [`hidden` option](#hidden-option)
+    - [Examples](#examples)
```

Comment:
Can we use `**Examples**` instead of `#### Examples` to avoid generating TOC entry for example?

Reply:
sure